CHANGELOG

```diff
@@ -8,6 +8,7 @@
 * Stripe Payment Intents: Allow cross_border_classification parameter [fatcatt316] #3508
 * EBANX: Fix `scrub` [chinhle23] #3521
 * Worldpay: Add `riskData` GSF [fatcatt316] #3514
+* Stripe Payment Intents: Revert "Allow cross_border_classification parameter" [fatcatt316] #3524
```

Comment:
Since these are released on minor version updates, what do you think about just removing line 8 above instead of having the changelog say "Add X, remove X"? The history of this being added and removed will be preserved by git.

Reply:
@Carrigan I agree! I thought about that some, too, and will go ahead and make that change now